t/client.t

```diff
@@ -38,9 +38,22 @@ my $client = OpenAPI::Client->new('data://main/test.json', app => app);
 my ($obj, $tx);
 
 is +ref($client), 'OpenAPI::Client::main_test_json', 'generated class';
+
 isa_ok($client, 'OpenAPI::Client');
 can_ok($client, 'addPet');
 
+subtest 'pre-mixing roles' => sub {
```

Comment:
This test must be guarded, since Role::Tiny might not be installed. Also, when was that package syntax introduced? If it's before 5.16, then it must be rewritten to keep the tests from breaking.

Reply:
Can you also update https://github.com/jhthorsen/openapi-client/blob/main/.github/workflows/ci-ubuntu.yml#L25 to install Role::Tiny?

Reply:
Actually, I think i'll just do a test w/ normal inheritance, b/c Roles shouldn't work any differently